nifi-nar-bundles/nifi-azure-bundle/pom.xml

```
@@ -31,7 +31,7 @@
         <azure.identity.version>1.4.5</azure.identity.version>
         <!-- azure-identity depends on msal4j transitively, keep these versions consistent -->
         <msal4j.version>1.11.0</msal4j.version>
-        <azure-cosmos.version>4.26.0</azure-cosmos.version>
+        <azure-cosmos.version>4.31.0</azure-cosmos.version>
```

The latest version appears to be 4.32.0, released on 2022-06-28. Can you upgrade to that version?
```suggestion
        <azure-cosmos.version>4.32.0</azure-cosmos.version>
```

---

@exceptionfactory will do. It looks like 4.32.1 is still vulnerable to [CVE-2020-8908](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8908) So maybe the best thing to do is hold off on this until they fix that issue?

---

If so, we can close this issue and wait for the update. I believe that might be the best option here.

---

@mr1716 NiFi uses the Jackson Bill-of-Materials dependency in the root Maven configuration to force the latest version across all modules. Based on that approach, NiFi is not subject to the vulnerability. However, it is still worth upgrading this dependency.

---

@exceptionfactory Thanks for the information. I'll come back to this in a bit. There are a few other updates that I want to test out first that go from >1 CVE to 0 CVE and see how that works.